Sources/App/routes.swift

```diff
@@ -12,9 +12,16 @@ public func routes(
     }
 
     commands.forEach { command in
-        router.post(command.name) { req in
-            try attempt {
-                try slack.handle(command: command, on: req)
+        router.post(command.name) { req -> Future<Response> in
+            do {
+                return try attempt {
+                    try slack.handle(command: command, on: req)
+                }
+            } catch {
+                return try SlackResponse(
+                    error.localizedDescription,
```

Comment:
Is this enough in order to have the `var reason` of a `SlackService.Error` be called? (Shouldn't our `SlackService.Error` be a `LocalizedError` instead of just a `Swift.Error` btw?) Or shouldn't we rename reason into description? Or is this all something that's magically handled by Vapor internals?

Reply:
It is magically handled by Vapor via `Debuggable` protocol, so when app is running in debug mode we will see more info. In release mode our `ThrowError` overrides `errorDescription` to debug info (source location of error)